plugins/bcc-login/includes/class-bcc-login-visibility.php

```diff
@@ -440,11 +508,10 @@ function on_render_block( $block_content, $block ) {
 
             $user_groups = $this->get_current_user_groups();
 
-
             if (!$user_groups) {
                 return '';
             }
-            if(count(array_intersect($block_groups, $user_groups)) == 0) {
+            if (count(array_intersect($block_groups, $user_groups)) == 0) {
```

Comment:
SHould we add a check for the full access groups here too?

Reply:
Very good point! Thank you!